spec/helpers.lua

```diff
@@ -10,6 +10,7 @@ local pl_file = require "pl.file"
 local pl_dir = require "pl.dir"
 local http = require "resty.http"
 local log = require "kong.cmd.utils.log"
+local cjson = require "cjson.safe"
```

Comment:
Trying to keep these ordered in *all* our files now. Looks like it should be moved to line 11. I know this can sound silly but so far all our new files follow this rule, so let's not already break it.